contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -322,6 +335,22 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
         );
     }
 
+    /**
+     * @notice Stakes a users vault shares
+     * @param numShares is the number of shares to stake
+     */
+    function stake(uint256 numShares) external nonReentrant {
+        require(stakingRewards != address(0)); // Removed revert msgs due to contract size limit
```

Comment:
Can cache `stakingRewards` since it's accessed twice

Reply:
Just added caching: https://github.com/ribbon-finance/ribbon-v2/pull/194/commits/b922790fc59b18311f214817bb237acb1b88d9b3